mlflow/store/tracking/rest_store.py

```diff
@@ -240,20 +240,43 @@ def delete_tag(self, run_id, key):
         req_body = message_to_json(DeleteTag(run_id=run_id, key=key))
         self._call_endpoint(DeleteTag, req_body)
 
-    def get_metric_history(self, run_id, metric_key):
+    def get_metric_history(self, run_id, metric_key, max_results=None, page_token=None):
         """
         Return all logged values for a given metric.
 
         :param run_id: Unique identifier for run
         :param metric_key: Metric name within the run
-
-        :return: A list of :py:class:`mlflow.entities.Metric` entities if logged, else empty list
+        :param max_results: Maximum number of metric history events (steps) to return per paged
+            query. Only supported in 'databricks' backend.
+        :param page_token: A Token specifying the next paginated set of results of metric history.
+
+        :return: A PagedList of :py:class:`mlflow.entities.Metric` entities if a paginated request
+            is made by setting ``max_results`` to a value other than ``None``, a List of
+            :py:class:`mlflow.entities.Metric` entities if ``max_results`` is None, else, if no
+            metrics of the ``metric_key`` have been logged to the ``run_id``, an empty list.
         """
         req_body = message_to_json(
-            GetMetricHistory(run_uuid=run_id, run_id=run_id, metric_key=metric_key)
+            GetMetricHistory(
+                run_uuid=run_id,
+                run_id=run_id,
+                metric_key=metric_key,
+                max_results=max_results,
+                page_token=page_token,
+            )
         )
         response_proto = self._call_endpoint(GetMetricHistory, req_body)
-        return [Metric.from_proto(metric) for metric in response_proto.metrics]
+
+        metric_history = [Metric.from_proto(metric) for metric in response_proto.metrics]
+        if max_results is not None:
+            if response_proto.next_page_token:
+                next_page_token = response_proto.next_page_token
+                return PagedList(metric_history, next_page_token)
+            else:
+                return PagedList(metric_history, None)
```

Comment:
```suggestion
            return PagedList(metric_history, response_proto.next_page_token)
```

Can we just return `PagedList(metric_history, response_proto.next_page_token)`?

Reply:
> the proto return is ""

I forgot about this. Can we add `or None`?

```python
return PagedList(metric_history, response_proto.next_page_token or None)
```